gobblin-runtime/src/main/java/gobblin/runtime/mapreduce/CliMRJobLauncher.java

```diff
@@ -41,33 +40,33 @@
  */
 public class CliMRJobLauncher extends Configured implements Tool {
 
-  private final Properties sysConfig;
-  private final Properties jobConfig;
+  private static final Logger LOG = LoggerFactory.getLogger(CliMRJobLauncher.class);
+  
+  private final Properties jobProperties;
 
-  public CliMRJobLauncher(Properties sysConfig, Properties jobConfig)
-      throws Exception {
-    this.sysConfig = sysConfig;
-    this.jobConfig = jobConfig;
+  public CliMRJobLauncher(Properties jobProperties) throws Exception {
+    this.jobProperties = jobProperties;
   }
 
   @Override
-  public int run(String[] args)
-      throws Exception {
-    final Properties jobProps = JobConfigurationUtils.combineSysAndJobProperties(this.sysConfig, this.jobConfig);
+  public int run(String[] args) throws Exception {
 
     Closer closer = Closer.create();
     try {
-      closer.register(new MRJobLauncher(jobProps, getConf())).launchJob(null);
-    } catch (JobException je) {
-      System.err.println("Failed to launch the job due to the following exception:");
-      System.err.println(je.toString());
+      try {
```

Comment:
Since Gobblin is on Java 7 now you can use `try-with-resources` instead of Guava's `Closer`. This should also allow you to avoid the nested `try-catch`.